org.adempiere.base.process/src/org/adempiere/process/InvoiceGenerateRMA.java

```diff
@@ -74,11 +74,14 @@ else if (name.equals("DocAction"))
                 log.log(Level.SEVERE, "Unknown Parameter: " + name);
         }
         
-        m_dateinvoiced = Env.getContextAsDate(getCtx(), Env.DATE);
-        if (m_dateinvoiced == null)
-        {
-            m_dateinvoiced = new Timestamp(System.currentTimeMillis());
+        if (m_dateinvoiced == null) {
+        	m_dateinvoiced = Env.getContextAsDate(getCtx(), Env.DATE);
+	        if (m_dateinvoiced == null)
+	        {
+	        	m_dateinvoiced = new Timestamp(System.currentTimeMillis());
+	        }
         }
+        p_Selection = getProcessInfo().getAD_InfoWindow_ID() > 0;

```

Comment:
to keep backward compatiblity, replace this with `if (getProcessInfo().getAD_InfoWindow_ID() > 0) p_Selection=true`